heka/cmd/heka-s3cat/main.go

```diff
@@ -0,0 +1,149 @@
+/***** BEGIN LICENSE BLOCK *****
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# The Initial Developer of the Original Code is the Mozilla Foundation.
+# Portions created by the Initial Developer are Copyright (C) 2014
```

Comment:
I'll remove them all, since they're all over the place.  I'll leave the MPL blurb in though, right?